port/webhook/schema.go

```diff
@@ -0,0 +1,150 @@
+package webhook
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func WebhookSchema() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"id": schema.StringAttribute{
+			Computed: true,
+		},
+		"identifier": schema.StringAttribute{
+			MarkdownDescription: "The identifier of the webhook",
+			Optional:            true,
+			Computed:            true,
+		},
+		"title": schema.StringAttribute{
+			MarkdownDescription: "The title of the webhook",
+			Optional:            true,
+		},
+		"icon": schema.StringAttribute{
+			MarkdownDescription: "The icon of the webhook",
+			Optional:            true,
+		},
+		"description": schema.StringAttribute{
+			MarkdownDescription: "The description of the webhook",
+			Optional:            true,
+		},
+		"enabled": schema.BoolAttribute{
+			MarkdownDescription: "Whether the webhook is enabled",
+			Optional:            true,
+			Computed:            true,
+			Default:             booldefault.StaticBool(false),
+		},
+		"security": schema.SingleNestedAttribute{
+			MarkdownDescription: "The security of the webhook",
+			Optional:            true,
+			Attributes: map[string]schema.Attribute{
+				"secret": schema.StringAttribute{
+					MarkdownDescription: "The secret of the webhook",
```

Comment:
feels kind of redundunt and dry.

maybe add a function to generate the markdownDescription from the property's name?